build/templates/_library_interpreter.py/numpy_write_method.py.mako

```diff
@@ -5,8 +5,8 @@
     import build.helper as helper
 
     parameters = f['parameters']
-    param_names_method = helper.get_params_snippet(f, helper.ParameterUsageOptions.LIBRARY_METHOD_DECLARATION)
-    param_names_library = helper.get_params_snippet(f, helper.ParameterUsageOptions.CTYPES_CALL)
+    param_names_method = helper.get_params_snippet(f, helper.ParameterUsageOptions.LIBRARY_INTERPRETER_METHOD_DECLARATION)
```

Comment:
In the `numpy_read` script above this was `LIBRARY_INTERPRETER_NUMPY_INTO_METHOD_DECLARATION`

Reply:
Yep. The [existing version](https://github.com/ni/nimi-python/blob/master/build/templates/session.py/numpy_write_method.py.mako) of the numpy_write file also uses non-NUMPY declaration/call parameters.